adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -856,7 +856,7 @@ public void testRefreshTokenPositive() throws IOException, InterruptedException,
 
         final String response2 = "{\"id_token\":\""
                 + TEST_IDTOKEN
-                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"10\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
+                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"3600\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
```

Comment:
Changing it to one hour because of the AT expires in one hour, I agree and I'm totally fine with it. 
I disagree to change any existing tests setup because "It causes my debugging to fail".